k8s.jsonnet

```
@@ -232,8 +241,15 @@
             },
           },
         ],
+        [if ($.config.ingress_tls) then 'tls']: [
```

`()` aren't required around if expressions
```suggestion
        [if $.config.ingress_tls then 'tls']: [
```